.gitignore

```diff
@@ -15,3 +15,4 @@ src/c/test/smoketest2
 src/c/test/smoketest2.o
 src/c/weaver/build/
 src/python/test/test-results/
+src/python/test/trash*/
```

Comment:
It would be better to combine tis commit w/ another change to the same file.

Reply:
both gitignore changes are in this commit /[4a32a9e](https://github.com/flux-framework/PerfFlowAspect/pull/45/commits/4a32a9e5c916615287f120ba3ef6066663134fd3)